client/src/components/Footer.css

```diff
@@ -0,0 +1,10 @@
+.Footer {
+  display: flex;
+  justify-content: center;
+  margin: 0;
```

Comment:
you have margin: 0 in your index.css so don't need it here. because it is cascading through all elements.